exporters/prometheus/src/exporter_utils.cc

```diff
@@ -252,6 +289,43 @@ prometheus_client::MetricType PrometheusExporterUtils::TranslateType(
   }
 }
 
+void PrometheusExporterUtils::SetTarget(const sdk::metrics::ResourceMetrics &data,
+                                        std::vector<::prometheus::MetricFamily> *output)
+{
+  if (output == nullptr || data.resource_ == nullptr)
+  {
+    return;
+  }
+  if (data.scope_metric_data_.empty())
+  {
+    return;
+  }
+  if ((*data.scope_metric_data_.begin()).metric_data_.empty())
+  {
+    return;
+  }
```

Comment:
Why do we skip target_info if scope_metrics is empty?  We should still write target_info even if we don't have any other metrics.

Reply:
Thanks and done.